sql/transactions/transactions.go

```diff
@@ -449,3 +340,47 @@ func AddResult(db *sql.Tx, id types.TransactionID, rst *types.TransactionResult)
 	}
 	return nil
 }
+
+// TransactionInProposal returns lowest layer of the proposal where tx is included after the specified layer.
+func TransactionInProposal(db sql.Executor, id types.TransactionID, after types.LayerID) (types.LayerID, error) {
+	var rst types.LayerID
+	rows, err := db.Exec("select layer from proposal_transactions where tid = ?1 and layer > ?2 order by layer asc limit 1",
```

Comment:
does min(layer) work here?

Reply:
i guess it does, but i wanted it to be consistent with query in TransactionInBlock. and there i would have to use group by block, which would be more complex if compared with current version